src/Microsoft.Data.SqlClient/tests/ManualTests/Microsoft.Data.SqlClient.ManualTesting.Tests.csproj

```diff
@@ -265,6 +265,10 @@
   </ItemGroup>
   <ItemGroup Condition="'$(TargetGroup)' == 'netfx'">
     <Reference Include="System.Transactions" />
+    <Compile Include="SQL\ConnectionPoolTest\TransactionPoolTest.cs" />
+  </ItemGroup>
+  <ItemGroup Condition="$(Configuration.Contains('Debug'))">
+    <Compile Include="SQL\ConnectionPoolTest\ConnectionPoolTest.Debug.cs" />
```

Comment:
Can it be prepared/reworked to enable testing in pipelines?
Maybe later if not now?

Reply:
It uses some code in `#if DEBUG` blocks, namely `KillConnection()` in `SNIHandle`. We can remove the #if directives if it's worth it.

Reply:
I think it will be better to start running these tests in "Release" configuration - even if it involves force killing connection.